common/djangoapps/student/models.py

```diff
@@ -812,6 +816,16 @@ def user_post_save_callback(sender, **kwargs):
                         cea.save()
                     continue
 
+                # Skip auto enrollment of user if
+                # DISABLE_AUTO_ENROLL_AFTER_ENROLLMENT_END is set to true
+                # and the course enrollment has ended
```

Comment:
```suggestion
                # Skip auto enrollment of user if # DISABLE_AUTO_ENROLL_AFTER_ENROLLMENT_END is set to true
                # and the course enrollment has ended
```

This formatting looks odd.

Reply:
Changed the wording since I have changed the logic here a bit.